spec/support/init_fixtures.rb

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2013 whiteleaf. All rights reserved.
+#
+
+#
+# テスト用にデータベース等を設定する
+#
+
+require "time"
+require "narou"
+require "database"
+
+Narou::LOCAL_SETTING_DIR_NAME.replace(".test_dot_narou")
+Narou::GLOBAL_SETTING_DIR_NAME.replace(".test_dot_narousetting")
+Database::ARCHIVE_ROOT_DIR_PATH.replace(".test_novel_data/")
+
+Narou.flush_cache
+
+def install_fixtures
+  root_dir = File.expand_path("../..", __dir__)
+  local_dir = File.join(root_dir, ".test_dot_narou")
+  global_dir = File.join(root_dir, ".test_dot_narousetting")
```

Comment:
__[rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `global_dir`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)